cc/utils/cose/cose.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2024 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CC_UTILS_COSE_COSE_H_
+#define CC_UTILS_COSE_COSE_H_
+
+#include <memory>
+#include <string>
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+#include "libcppbor/include/cppbor/cppbor.h"
+#include "libcppbor/include/cppbor/cppbor_parse.h"
+
+namespace oak::utils::cose {
+
+// COSE_Sign1 object.
+// <https://datatracker.ietf.org/doc/html/rfc8152#section-4.2>
+struct CoseSign1 {
```

Comment:
These should be `class`-es, not `struct`-s: https://google.github.io/styleguide/cppguide.html#Structs_vs._Classes

Reply:
Done